### PR TITLE
Editorial pass on 5.0 release notes

### PR DESCRIPTION
**Pull request by ahankinson**
This PR contains a review of the text for the 5.0 release. The content and intent was preserved, but some formatting and wording was modified.

Specifically:
 - Long URLs were shortened with linked text to help the prose flow better. A list of repository URLs was added instead of inline text.
 - The MEI Basic section was slightly expanded
 - A paragraph was added about the IG additions to the guidelines
 - Wording was simplified

### CODE REVIEW

**Review by rettinghaus — APPROVED** [review]
Thank you, @ahankinson.

File: source/docs/01-introduction.xml [hunk 1]
```diff
@@ -179,42 +179,54 @@
...
-                     complexities moves from generic tools to each project, which should know best how 
-                     their internal data model maps to a shared profile such as MEI Basic. MEI Basic 
-                     is a strict subset of MEI, intentionally starting with a relatively small footprint 
-                     of supported features. However, the subset of included features may be expanded if
-                     need be.</p>
+                     <p>As a framework to encode music, MEI offers extensive flexibility to encode music
+                     documents of various kinds, and for a wide variety of uses. For scholarly research, 
+                     this flexibility is necessary and is one of the greatest strengths of MEI. At the 
+                     same time, we recognize that this flexibility presents challenges for broad adoption 
+                     of MEI as a notation interchange format. For developers providing "full" MEI support
```
Comment (ahankinson) [inline source/docs/01-introduction.xml]:
```suggestion
                     of MEI as a notation interchange format. For developers, providing "full" MEI support 
```

**Review by bwbohl — APPROVED** [review]

**Comment by ahankinson** [comment]
Hmm @bwbohl are these being closed on purpose?

**Comment by bwbohl** [comment]
@ahankinson sorry, was a mistake…

**Pull request merged**